migration/1648085509369-addGivingBlockOrganizationToGivingBlockProjects.ts

```diff
@@ -0,0 +1,26 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class addGivingBlockOrganizationToGivingBlockProjects1648085509369
+  implements MigrationInterface
+{
+  // this is to fix newly created givingBlocksProjects after the older migration ran
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const projectTableExists = await queryRunner.hasTable('project');
+
+    if (!projectTableExists) return;
+
+    const givingBlockOrganization = (
+      await queryRunner.query(`SELECT * FROM organization
+              WHERE label='givingBlock'`)
+    )[0];
+
+    if (!givingBlockOrganization) return;
+    await queryRunner.query(`
+            UPDATE project
+            SET "organizationId" = ${givingBlockOrganization.id}
+            WHERE "givingBlocksId" IS NOT NULL
+        `);
+  }
```

Comment:
I think this is redundant because we related givingBlocks projects to organization before in this migration file

https://github.com/Giveth/impact-graph/blob/staging/migration/1646306281286-relateExistingProjectsToOrganizations.ts

Reply:
Oh got it, you are absolutely right